packages/grpc-js/src/logging.ts

```diff
@@ -18,7 +18,7 @@
 import { LogVerbosity } from './constants';
 
 let _logger: Partial<Console> = console;
-let _logVerbosity: LogVerbosity = LogVerbosity.DEBUG;
+let _logVerbosity: LogVerbosity = LogVerbosity.ERROR;
```

Comment:
Does this match the default verbosity of the compiled addon?

Reply:
Yes, I'm pretty sure it does.